2-mandatory/3-financial-times.js

```diff
@@ -15,15 +23,49 @@ function potentialHeadlines(allArticleTitles) {
 */
 function titleWithFewestWords(allArticleTitles) {
     // TODO
+    let titleLength = [];
+    for (let i = 0; i < allArticleTitles.length; i++){
+        let titleArray = allArticleTitles[i].split(" ");
+        titleLength.push(titleArray.length);
+    }
+    const minLength = Math.min(...titleLength);
+    const minIndex = titleLength.indexOf(minLength);
+    return allArticleTitles[minIndex];
 }
 
```

Comment:
Very good solution, very similar to mine

Reply:
Nice to hear that.